Autopilot/Src/altimeter.cpp

```diff
@@ -0,0 +1,136 @@
+#include "altimeter.hpp"
+
+#include "stm32f7xx_hal.h"
+#include "stm32f7xx_hal_i2c.h"
+
+#include <algorithm>
+
+/***********************************************************************************************************************
+ * Definitions
+ **********************************************************************************************************************/
+
+#define I2C_BUS 1
+#define LEFT_SHIFTED_ALTIMETER_SLAVE_ADDRESS 0xC0 // actual slave address is 0x60. But for whatever reason, the cubeMX drivers don't left shift the address so I have to do that myself
+#define ALT_CTRL_REG1 0x26
+#define ALT_OUT_P_MSB_REG 0x01
+#define STANDBY_MODE_8_OS 0x98
+#define ACTIVE_MODE_8_OS 0x99
+
+#define MEM_SIZE_1_B 1
+
+/***********************************************************************************************************************
+ * Static variables
+ **********************************************************************************************************************/
+
+static bool dataIsNew;
```

Comment:
`= false`

Reply:
initialized to false in the constructor.